client/__tests__/dummy-react-base-test.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
```

Comment:
I added this dummy test to check that all is good with RTL - I can remove it once the PR is approved (or leave it, whatever you prefer)

Reply:
If we merge #1702 first before this I can write a basic smoketest for the Stripe settings page. Would be nice to have anyways.

Reply:
Thanks for doing this! Once this is merged I can actually write a smoke test for #1702 and then delete the dummy test. 👍